src/client/utils/tracking/adtraction.ts

```diff
@@ -15,7 +15,12 @@ import { NoComboTypes, DkBundleTypes } from './tracking'
 // Exclude single contract types that are only sold as part of bundles
 type TypeOfContractExcludedUnused = Exclude<
   TypeOfContract | NoComboTypes | DkBundleTypes,
-  'DK_ACCIDENT' | 'DK_ACCIDENT_STUDENT' | 'DK_TRAVEL' | 'DK_TRAVEL_STUDENT'
+  | 'DK_ACCIDENT'
+  | 'DK_ACCIDENT_STUDENT'
+  | 'DK_TRAVEL'
+  | 'DK_TRAVEL_STUDENT'
+  | TypeOfContract.SeAccident
+  | TypeOfContract.SeAccidentStudent
```

Comment:
Maybe be consistent here and either use strings for `TypeOfContract.SeAccident` and `TypeOfContract.SeAccidentStudent` as well, or replace the strings `'DK_ACCIDENT'` and so on with that syntax which explicitly references the enum 🤔